indico/modules/events/registration/badges.py

```diff
@@ -87,7 +87,12 @@ def _draw_badge(self, canvas, registration, template, tpl_data, pos_x, pos_y):
 
         placeholders = get_placeholders('designer-fields')
 
-        for item in tpl_data.items:
+        # Print images first
+        images = [key for key in placeholders if hasattr(placeholders[key], 'is_image')]
```

Comment:
this won't work since all placeholders have it

Reply:
Correct. I'm going to fix it:

`images = [key for key in placeholders if placeholders[key].is_image]`

Reply:
please see my line in the comment thread just below ;) it's a bit cleaner